docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -37,236 +37,6 @@ For information about multitenant support, see xref:security-openid-connect-mult
-For more information, see xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints].
-
-=== Starting and Configuring the Keycloak Server
-
-To start a Keycloak Server you can use Docker and just run the following command:
-
-[source,bash,subs=attributes+]
-----
-docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8180:8080 quay.io/keycloak/keycloak:{keycloak.version} start-dev
-----
-
-where `keycloak.version` should be set to `17.0.0` or higher.
-
-You should be able to access your Keycloak Server at http://localhost:8180[localhost:8180].
-
-Log in as the `admin` user to access the Keycloak Administration Console. Username should be `admin` and password `admin`.
-
-Import the {quickstarts-tree-url}/security-openid-connect-web-authentication-quickstart/config/quarkus-realm.json[realm configuration file] to create a new realm. For more details, see the Keycloak documentation about how to https://www.keycloak.org/docs/latest/server_admin/index.html#_create-realm[create a new realm].
-
-=== Running the Application in Dev and JVM modes
-
-To run the application in a dev mode, use:
-
-include::{includes}/devtools/dev.adoc[]
-
-When you're done playing with dev mode you can run it as a standard Java application.
-
-First compile it:
-
-include::{includes}/devtools/build.adoc[]
-
-Then run it:
-
-[source,bash]
-----
-java -jar target/quarkus-app/quarkus-run.jar
-----
-
-=== Running the Application in Native Mode
-
-This same demo can be compiled into native code: no modifications required.
-
-This implies that you no longer need to install a JVM on your
-production environment, as the runtime technology is included in
-the produced binary, and optimized to run with minimal resource overhead.
-
-Compilation will take a bit longer, so this step is disabled by default;
-let's build again by enabling the native build:
-
-include::{includes}/devtools/build-native.adoc[]
-
-After getting a cup of coffee, you'll be able to run this binary directly:
-
-[source,bash]
-----
-./target/security-openid-connect-web-authentication-quickstart-runner
-----
-
-=== Testing the Application
-
-To test the application, you should open your browser and access the following URL:
-
-* http://localhost:8080/tokens[http://localhost:8080/tokens]
-
-If everything is working as expected, you should be redirected to the Keycloak server to authenticate.
-
-In order to authenticate to the application you should type the following credentials when at the Keycloak login page:
-
-* Username: *alice*
-* Password: *alice*
-
-After clicking the `Login` button you should be redirected back to the application.
-
-Please also see the <<integration-testing-keycloak-devservices, Dev Services for Keycloak>> section below about writing the integration tests which depend on `Dev Services for Keycloak`.
-
 == Reference Guide
```

Comment:
Probably a comment that is outside the scope of this PR restructure:

```suggestion
== References
```
Made me wonder if the doc _templates are missing an optional **References** section? Perhaps one for the Contributing guide...